be/src/runtime/fragment_mgr.cpp

```diff
@@ -266,6 +266,14 @@ void FragmentMgr::coordinator_callback(const ReportStatusRequest& req) {
             }
 
             if (enable_profile) {
+                DCHECK(req.profile != nullptr);
+                TDetailedReportParams detailed_param;
+                detailed_param.__isset.fragment_instance_id = false;
+                detailed_param.__isset.profile = true;
+                detailed_param.__isset.loadChannelProfile = false;
```

Comment:
why loadChannelProfile is always false?

Reply:
Because load profile reporting will not reach here.